account_analytic_distribution_to_m2o/models/account_move_line.py

```diff
@@ -0,0 +1,34 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import api, fields, models
+import json
+
+class AccountMoveLine(models.Model):
+    _inherit = 'account.move.line'
+
+    first_analytic_account_id = fields.Many2one(
+        'account.analytic.account',
+        string="Analytic Account",
+        compute='_compute_first_analytic_account_id',
+        index=True,
+        help="Returns the main analytic account"
```

Comment:
Rename to be coherent with below information:
```suggestion
        help="Returns the first analytic account saved at analytic distribution"
```